server/routes/contacts/create/enter-dob/createContactEnterDobController.ts

```diff
@@ -9,7 +9,14 @@ export default class CreateContactEnterDobController implements PageHandler {
   GET = async (req: Request, res: Response): Promise<void> => {
     const { journeyId } = req.params
     const journey = req.session.createContactJourneys[journeyId]
-    res.render('pages/contacts/create/enterDob', { journey })
+    const view = {
+      journey,
+      isKnown: res.locals?.formResponses?.isKnown ?? journey?.dateOfBirth?.isKnown,
+      day: res.locals?.formResponses?.day ?? journey?.dateOfBirth?.day,
+      month: res.locals?.formResponses?.month ?? journey?.dateOfBirth?.month,
+      year: res.locals?.formResponses?.year ?? journey?.dateOfBirth?.year,
+    }
```

Comment:
Minor only - if we adopt this method (building the viewModel for forms in controllers/handlers) maybe consistency of its name (view, viewModel)?